### ISSUE TITLE
TypeError: cannot concatenate 'str' and 'NoneType' objects

### ISSUE THREAD

**Issue by balzanelli** [emesene/emesene#141]
```
Traceback (most recent call last):
  File "/usr/lib/pymodules/python2.6/papyon/client.py", line 368, in property_changed
    self._dispatch(method_name, contact)
  File "/usr/lib/pymodules/python2.6/papyon/event/__init__.py", line 44, in _dispatch
    if event_handler._dispatch_event(name, *args):
  File "/usr/lib/pymodules/python2.6/papyon/event/__init__.py", line 65, in _dispatch_event
    handler(*params)
  File "/usr/share/emesene/emesene/e3/papylib/PapyEvents.py", line 149, in on_contact_msn_object_changed
    self._client._on_contact_msnobject_changed(contact)
  File "/usr/share/emesene/emesene/e3/papylib/Worker.py", line 607, in _on_contact_msnobject_changed
    (download_ok, download_failed))
  File "/usr/lib/pymodules/python2.6/papyon/p2p.py", line 334, in request
    session.invite()
  File "/usr/lib/pymodules/python2.6/papyon/p2p.py", line 587, in invite
    session.invite()
  File "/usr/lib/pymodules/python2.6/papyon/msnp2p/msnobject.py", line 64, in invite
    self._invite(self._context)
  File "/usr/lib/pymodules/python2.6/papyon/msnp2p/session.py", line 156, in _invite
    self._send_slp_message(message)
  File "/usr/lib/pymodules/python2.6/papyon/msnp2p/session.py", line 261, in _send_slp_message
    self._application_id, message)
  File "/usr/lib/pymodules/python2.6/papyon/msnp2p/transport/transport_manager.py", line 127, in send_slp_message
    self.send_data(peer, peer_guid, application_id, 0, str(message))
  File "/usr/lib/pymodules/python2.6/papyon/msnp2p/transport/transport_manager.py", line 132, in send_data
    transport.send(peer, peer_guid, blob)
  File "/usr/lib/pymodules/python2.6/papyon/msnp2p/transport/base.py", line 103, in send
    self._process_send_queue()
  File "/usr/lib/pymodules/python2.6/papyon/msnp2p/transport/base.py", line 206, in _process_send_queue
    self.__send_chunk(peer, peer_guid, chunk)
  File "/usr/lib/pymodules/python2.6/papyon/msnp2p/transport/base.py", line 224, in __send_chunk
    self._send_chunk(peer, peer_guid, chunk)
  File "/usr/lib/pymodules/python2.6/papyon/msnp2p/transport/switchboard.py", line 102, in _send_chunk
    peer_guid + "}"}
TypeError: cannot concatenate 'str' and 'NoneType' objects
```

**Comment by c10ud** [comment]
emesene and papyon version?

**Comment by c10ud** [comment]
this bug has been fixed with papyon version 0.5.2, i do find strange you've been able to connect with that version since i should have blocked it from loading. anyway upgrading your papyon will do the trick. (just cd to emesene2 dir and git submodule init && git submodule update. this will locally fetch a copy of latest papyon git master only for emesene)
